mobile/package.json

```diff
@@ -0,0 +1,63 @@
+{
+  "name": "mobile",
+  "version": "0.1.0",
```

Comment:
I don't think this should be versioned, or else we should add versions to the other examples.